src/organisation/containers/organisation/organisation.component.ts

```diff
@@ -33,13 +39,19 @@ export class OrganisationComponent implements OnInit, OnDestroy {
   constructor(
     private readonly orgStore: Store<fromStore.OrganisationState>,
     private readonly authStore: Store<fromAuthStore.AuthState>,
+    private readonly lovRefDataService: LovRefDataService,
     private readonly featureToggleService: FeatureToggleService
   ) {
     this.getOrganisationDetailsFromStore();
   }
 
   ngOnInit(): void {
     this.isFeatureEnabled$ = this.featureToggleService.getValue(AppConstants.FEATURE_NAMES.newRegisterOrg, false);
+
+    this.lovRefDataService.getListOfValues(this.CATEGORY_ORGANISATION_TYPE, true).subscribe((orgTypes) => {
```

Comment:
@connorpgpmcelroy - would be good to assign this to some variable that can later be unsubscribed in ngOnDestroy, to avoid any resource leaks